Rubberduck.CodeAnalysis/CodePathAnalysis/Nodes/Abstract/NodeBase.cs

```diff
@@ -2,6 +2,7 @@
 using System.Collections.Immutable;
 using Antlr4.Runtime.Tree;
 using Rubberduck.Parsing.Symbols;
+using Rubberduck.VBEditor;
```

Comment:
Do we actually need that?

Reply:
@bclothier not really, but the `HashCode.Compute` method is handy.

Reply:
Can we move it into `JunkDrawer` instead? That way you don't need a reference to the `VBEditor`, which we want to avoid exposing toxic wastes stored in that project. ;-)

Reply:
That's actually a more sensible place to put it than `VBEditor`

Reply:
@bclothier it actually impacts a lot of stuff I that don't feel is right to shuffle around with *this* PR though.

Reply:
In this case, we can just mark it off with a `TODO`. Besides, I'm not sure `JunkDrawer` can stay `JunkDrawer` for very much if it's filling up with shared methods that spans projects. 😆